kubenow/gluster-s3/README.md

```diff
@@ -0,0 +1,44 @@
+Gluster-S3 Helm Chart
+====================
+
+This chart can be used to deploy Gluster-S3, a S3 object storage interface for GlusterFS.
+
+Helm chart settings and deployment
+----------------------------------
+
+The following tables lists the configurable parameters of the chart and their default values.
+
+| Parameter                    | Description           | Default                 |
+|------------------------------|-----------------------|-------------------------|
+| `image`                      | Default image name    | `gluster/gluster-object`|
+| `imageTag`                   | Default image tag     | `latest`                |
+| `imagePullPolicy`            | Default pull policy   | `IfNotPresent`          |
+| `secretName`                 | Secret name           | `s3secret`              |
+| `defaultBucket.enabled`      | S3 bucket creation    | `false`                 |
+| `defaultBucket.name`         | S3 bucket name        | `mybucket`              |
+| `pvc1Request`                | Main PVC request      | `120Gi`                 |
+| `pvc2Request`                | Meta PVC request      | `10Gi`                  |
+| `hostNetwork`                | How to expose the svc | `false`                 |
+
+
+This service can be deployed with default settings via Helm using a simple command. You must deploy by setting a S3 account, user and password. It is strongly recommended that these contain more than 6 characters.
+
+```console
+$ helm install --name s3-interface account=<s3-account>,user=<s3-user>,password=<s3-password> kubenow/gluster-s3
```

Comment:
account, user and password are not in the table, please make sure that all of the vaules are in the table even if null or empty string, so the user has a reference